---
title: About me
layout: default
---

<section>
I’m a postdoctoral researcher at the [University of Cambridge][] &
<abbr>[EMBL][]</abbr>-<abbr>[EBI][]</abbr>. I analyse genomic data to answer
biological questions. Here are some of the things I’ve been up to:
</section>

<section>
# Research

My current work is on the regulation of translation in mammals through changes
in <abbr>tRNA</abbr> abundance and codon usage. I am also looking at the
extended polymerase III transcriptome, in particular <abbr>SINE</abbr>s. My [PhD
thesis][] was titled *“Investigating the link between <abbr>tRNA</abbr> and
<abbr>mRNA</abbr> abundance in mammals”*.

Before this, I’ve worked in algorithms and software development, in particular
implementing high-throughput sequencing read mapping tools, and creating
frameworks for the automatic parallelisation of high-performance code on
multicore architecture. I have also done work in cell regulatory networks.
</section>

<section>
# Hacking

I’m the developer of ‹[modules][]›, an alternative [R][] code modularisation
system. It replaces conventional R packages and fixes [some of its
flaws][modules-rationale]. The mechanism is borrowed from Python packages and
JavaScript modules.

I developed the first version of the [LaTeX][] package ‹[minted][]›, which
provides code pretty-printing by wrapping [Pygments][].

I strive to make development with [C++][] less painful by showcasing the use of
clean, high-level code over low-level constructs. [C++11][] has made this
immensely easier.
</section>

<section>
# Teaching

I have, at various times, taught on the following subjects:

* High-throughput sequencing with focus on <abbr>RNA</abbr>-seq & ChIP-seq
  <small>(University of Cambridge)</small>
* Computer science 101/102 <small>(Freie Universität Berlin)</small>
* Algorithms in bioinformatics <small>(Freie Universität Berlin)</small>
* Database systems <small>(Freie Universität Berlin)</small>
* C++ <small>(Freie Universität Berlin)</small>
* VB <small>(ActiveVB workshop)</small>
* Unix/Bash <small>(<abbr>EMBL</abbr> workshop)</small>
* LaTeX <small>(<abbr>EMBL</abbr> workshop)</small>
* R <small>(<abbr>EMBL</abbr> workshop)</small>
* Git <small>(<abbr>EMBL</abbr> workshop)</small>
</section>

<section>
# Selected publications

* <span class="bib authors">B. M. Schmitt\*, K. L. M. Rudolph\*, P. Karagianni,
  N. A. Fonseca, R. J. White, I. Talianidis, D. T. Odom, J. C. Marioni, & C.
  Kutter</span>: [<span class="bib title">“High-resolution mapping of
  transcriptional dynamics across tissue development reveals a stable
  <abbr>mRNA</abbr>–<abbr>tRNA</abbr> interface”</span>][bib-1]. <span
  class="bib journal">Genome Res</span>, <span class="bib
  issue">24(11):1797–807</span> (<span class="bib date">Nov 2014</span>)

<small>\* indicates joint first authors</small>

For more, see my [Google Scholar profile][].
</section>

<section>
# CV

[You can download my <abbr>CV</abbr>][cv], or view the [source code][cv-source].
</section>

<section>
# Online

You can follow my activity online:

* On [Twitter][]
* On [Stack Overflow][]
* On [Github][]
</section>

[University of Cambridge]: http://www.cam.ac.uk/
[EMBL]: http://embl.org
[EBI]: http://www.ebi.ac.uk
[PhD thesis]: https://github.com/klmr/thesis
[R]: http://r-project.org
[modules]: https://github.com/klmr/modules
[modules-rationale]: https://github.com/klmr/modules/wiki/Design-rationale
[LaTeX]: https://www.latex-project.org/
[minted]: http://ctan.org/pkg/minted
[Pygments]: http://pygments.org/
[C++]: https://isocpp.org/
[C++11]: https://en.wikipedia.org/wiki/C%2B%2B11
[C++ for loop]: https://github.com/klmr/cpp-ranges
[Google Scholar profile]: https://scholar.google.com/citations?user=ALuSMe8AAAAJ&hl=en
[bib-1]: http://dx.doi.org/10.1101/gr.176784.114
[cv]: …
[cv-source]: https://github.com/klmr/cv
[Twitter]: http://twitter.com/klmr
[Stack Overflow]: stackoverflow.com/users/1968/konrad-rudolph
[Github]: https://github.com/klmr
